### PR TITLE
Fix various lay-out issues

### PR DESCRIPTION
**Pull request by tsteenbe**
Fixes the below issues:

1. List and links in 1.7.6 not rendered
2. Example tag-value in 4.16.5 is in red
3. Example RDF in 5.3.6 is in red
4. Classes in 5.3.6 are not in highlighed code
5. Reference to "Appendix IV" in 5.5.1 is not a link
7. Example RDF in 7.1.6 is incorrectly indented
8. Property and class in 8.2.6 not in highlighted as code
9. Sections 9.2 and 9.3 do not appear in table of contents
10. None of the sections of Appendix II appear in table of contents
11. Words in some code examples are bolded

Signed-off-by: Thomas Steenbergen <thomas.steenbergen@here.com>

### CODE REVIEW

File: chapters/1-rationale.md [hunk 1]
```diff
@@ -71,12 +71,12 @@ In an SPDX document, Relationship elements can be used to indicate relationships
...
     * The JSON Schema for SPDX can be found in the [SPDX Spec Git Repository Schema directory](https://github.com/spdx/spdx-spec/schemas/spdx-schema.json)
-* Resource Description Framework (RDF also referred to as RDF/XML) see: https://www.w3.org/TR/rdf-syntax-grammar/
-* `tag:value` flat text file as described in this specification
-* .xls spreadsheets
+* **Resource Description Framework** (RDF also referred to as RDF/XML) see: <https://www.w3.org/TR/rdf-syntax-grammar/>
+* **tag:value** flat text file as described in this specification
+* **.xls** spreadsheets
 
 In addition to the supported formats, the following format is in development with a plan to complete the specification by SPDX 3.0:
 * Extensible Markup Language (XML) see: https://www.w3.org/TR/2008/REC-xml-20081126/
```
Comment (swinslow) [inline chapters/1-rationale.md:82]: I think the XML line also needs a line break before the bullet, and asterisks for bolding. (I know the line break might not be needed for GitHub Markdown, but it seems to be needed for the generated spec bullet lists.)

File: chapters/3-package-information.md [hunk 2]
```diff
@@ -523,38 +673,45 @@ Example:
 
 Example:
 
-    PackageChecksum: SHA1: 85ed0817af83a24ad8da68c2b5094de69833983c
+```text
+PackageChecksum: SHA1: 85ed0817af83a24ad8da68c2b5094de69833983c
+```
+```text
```
Comment (swinslow) [inline chapters/3-package-information.md:679]: Does MarkDown need a line break between these two sections? I wasn't sure but I see there's an additional empty line for the other examples (e.g. between SHA256 and MD5 below).

File: chapters/3-package-information.md [hunk 3]
```diff
@@ -971,41 +1179,51 @@ where:
...
-            <spdx:ExternalRef>
-                <spdx:referenceCategory rdf:resource="spdx:referenceCategory_packageManager" />
-                <spdx:referenceType rdf:resource="http://spdx.org/rdf/refeferences/maven-central" />
-                <spdx:referenceLocator>org.apache.commons:commons-lang:3.2.1</spdx:referenceLocator>
-            </spdx:ExternalRef>
-        </spdx:externalRef>
-        ...
-    </spdx:package>
+```text
+<spdx:Package  rdf:about="...">
```
Comment (swinslow) [inline chapters/3-package-information.md:1199]: looks like there's an extra empty space between `Package` and `rdf`. I don't particularly care but I'm noting it as I go through these edits  :)

File: chapters/5-snippet-information.md [hunk 4]
```diff
@@ -326,22 +362,26 @@ In `tag:value` format multiple lines are delimited by `<text> .. </text>`.
...
 Example:
 
-    SnippetLicenseComments: <text>The concluded license was taken from package xyz, from which the snippet was copied into the current file.
-    The concluded license information was found in the COPYING.txt file in package xyz.</text>
+```text
+SnippetLicenseComments: <text>The concluded license was taken from package xyz, from which the snippet was copied into the current file.
+The concluded license information was found in the COPYING.txt file in package xyz.</text>
+```
 
 **5.7.6 ** RDF: Property `spdx:licenseComments` in class `spdx:Snippet`
```
Comment (swinslow) [inline chapters/5-snippet-information.md:370]: I think there's an extra space between `5.7.6` and the closing `**` tag -- it looks like that might be causing the rest of this section to render as bold? I'm guessing that space should be removed.

File: chapters/appendix-VI-external-repository-identifiers.md [hunk 5]
```diff
@@ -14,59 +14,68 @@ There are currently four defined categories:
...
-
 ## Security
 
 ### cpe22Type <a name="cpe22"></a>
 
 Locator Format:
 
-    "[c][pP][eE]:/[AHOaho]?(:[A-Za-z0-9\._\-~%]*){0,6}"
+```text
+"[c][pP][eE]:/[AHOaho]?(:[A-Za-z0-9\._\-~%]*){0,6}"
```
Comment (swinslow) [inline chapters/appendix-VI-external-repository-identifiers.md:24]: Should the surrounding quotation marks (") be removed from this? now that it is bordered by the backticks as well.

File: chapters/appendix-VI-external-repository-identifiers.md [hunk 6]
```diff
@@ -14,59 +14,68 @@ There are currently four defined categories:
...
-    (:(((\?*|\*?)([a­zA­Z0­9\­\._]|(\\[\\\*\?!
-    "#$$%&'\(\)\+,/:;<=>@\[\]\^`\{\|}~])
-    )+(\?*|\*?))|[\*\­])){5}
-    (:(([a­zA­Z]{2,3}(­([a­zA­Z]{2}|[0­9]{3
-    }))?)|[\*\­]))
-    (:(((\?*|\*?)([a­zA­Z0­9\­\._]|(\\[\\\*\?!
-    "#$$%&'\(\)\+,/:;<=>@\[\]\^`\{\|}~])
-    )+(\?*|\*?))|[\*\­])){4}"
+```text
+"cpe:2\.3:[aho\*\­]
```
Comment (swinslow) [inline chapters/appendix-VI-external-repository-identifiers.md:42]: Same question as above, for this regex should the start and end quotation marks (") be removed?

**Comment by kestewart** [comment]
Following up,  since we want to get most of these changes in, and a review cycle, and Thomas is hopefully zzz'ing.   We're going to merge this as is,  and @swinslow has offered to create a PR to fix the issues he's spotted.

**Comment by swinslow** [comment]
@kestewart #299 fixes the typos I noted in my comment above.

**Pull request merged**